src/assets/styles/readme.md

```diff
@@ -0,0 +1,66 @@
+# Calcite global styles
+
+When building any component in calcite-components or [calcite-app-components](https://github.com/esri/calcite-app-components), please use these set of global variables (CSS variables and Sass mixins or variables)
+
+## From calcite-components
+
+### Color
+
+[`_colors.scss`](https://github.com/Esri/calcite-components/blob/master/src/assets/styles/_colors.scss): you'll find the global CSS variables for both light and dark theme. 
+
+## From [calcite-base](https://github.com/esri/calcite-base)
+
+### Typography
+[typography mixins from calcite-base](https://github.com/Esri/calcite-base/blob/master/dist/_type.scss)
+
+#### Font-size
+
+Here are the most common use cases font-size (Sass):
+```
+@include font-size(1)
```

Comment:
Small thing: Might be good to show an example of the CSS that the mixin renders.